OrderDetailsView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  OrderDetailsView.swift
+//  Drawing
+//
+//  Created by Iulian Florea on 7/16/21.
+//
+
+import SwiftUI
+import URLImage
+
+struct OrderDetailsView: View {
+    let order: Order
+    var statusTypes = ["new", "pending", "delivered"]
+    @State var selected = Order.dummyData.status
```

Comment:
Why no ViewModel, like you used for OrderList? also the statusTypes could have been an enum 🤔